### DISCUSSION TITLE
Fetch updated "4.8-windowsservercore-ltsc2019" image with fixes of CVE-2021-24111

### DISCUSSION THREAD

**Discussion by vikasbhaimehta** [microsoft/dotnet-framework-docker#740] [General]
We recently found an high vulnerability [CVE-2021-24111] in image "4.8-20210209-windowsservercore-ltsc2019". As per the scan the fixes is available in KB4601887. So how can we get the updated image with this fixes?

Appreciate guidance in advance.

**Comment by mthalman** [comment]
There are two separate cumulative updates that both fix [CVE-2021-24111](https://msrc.microsoft.com/update-guide/vulnerability/CVE-2021-24111) on Windows Server 2019: [KB4601887](https://support.microsoft.com/topic/february-9-2021-kb4601887-cumulative-update-for-net-framework-3-5-4-7-2-and-4-8-for-windows-10-version-1809-and-windows-server-version-2019-3bc7f63e-6563-ff89-f499-cb3ffd9fb0cb) (for 3.5, 4.7.2, and 4.8) and [KBKB4601055](https://support.microsoft.com/topic/february-9-2021-kb4601055-cumulative-update-for-net-framework-3-5-and-4-8-for-windows-10-version-1809-and-windows-server-version-2019-38aea913-ccd7-bc76-db60-9521318f97cb) (for 3.5 and 4.8). As you can see, in both those links it states that it addresses CVE-2021-24111.  The `4.8-windowsservercore-ltsc2019` tag contains the latter KB number as you can see in the Dockerfile: https://github.com/microsoft/dotnet-framework-docker/blob/e3615cceaa30d74938241c6c0c81e762908a25af/src/runtime/4.8/windowsservercore-ltsc2019/Dockerfile#L19

So my guess is that this circumstance of having two cumulative updates that both fix the CVE is just confusing the scanner and leading to a false positive.  What scanning service are you using?
